web/text_layer_builder.js

```diff
@@ -31,6 +31,8 @@ import { renderTextLayer } from "pdfjs-lib";
  * @property {TextHighlighter} highlighter - Optional object that will handle
  *   highlighting text from the find controller.
  * @property {TextAccessibilityManager} [accessibilityManager]
+ * @property {boolean} [isOffscreenCanvasSupported] - Allows to use an
+ *   OffscreenCanvas if needed.
```

Comment:
Please don't forget to remove the now unused `textLayerDiv`-line above, and please update https://github.com/mozilla/pdf.js/blob/master/web/interfaces.js and https://github.com/mozilla/pdf.js/blob/master/web/default_factory.js accordingly as well.